src/Basis/Constructors.jl

```diff
@@ -2,28 +2,152 @@
 # finite element bases
 # ------------------------------------------------------------------------------
 
+using Polynomials
+
+# ------------------------------------------------------------------------------
+# conformal map functions for generating transformed polynomial bases
+# ------------------------------------------------------------------------------
+
+"""
+```julia
+sausage(d)
+```
+# Arguments:
```

Comment:
The suggestion interface isn't working right here, but we have an extra line between the 3 backticks for the function name and the `# Arguments:` list (same with other new functions below)